.env.example

```diff
@@ -49,10 +49,10 @@ SQS_SECRET_ACCESS_KEY=
 SQS_PREFIX=
 SQS_DEFAULT_QUEUE=
 
-NORTHSTAR_URL=https://northstar-qa.dosomething.org
-NORTHSTAR_AUTH_ID=
+NORTHSTAR_URL=https://identity-dev.dosomething.org
+NORTHSTAR_AUTH_ID=dev-oauth
 NORTHSTAR_AUTH_SECRET=
-NORTHSTAR_CLIENT_ID=
+NORTHSTAR_CLIENT_ID=dev-machine
```

Comment:
@DFurnes random q! Why is this `importer-client` in chompy but `dev-machine` here? I would have expected something related to Rogue?

Reply:
We should be using `dev-machine` on our locals (so that we can audit which environment requests or changes are coming from), and `importer-client` on Chompy (prod?).

Reply:
oooh ok good to know! I've been trying to rebuild my local chompy and still haven't gotten it to work so I'll do that this time and maybe that will do that trick. Thanks for explaining!